plugins/telemetry/telemetry.cpp

```diff
@@ -299,4 +299,19 @@ const char *Telemetry::result_str(Result result)
     }
 }
 
+bool operator==(const Telemetry::Position &lhs, const Telemetry::Position &rhs)
+{
+    return lhs.latitude_deg == rhs.latitude_deg
```

Comment:
Can we compare two doubles ?

Reply:
Good point. I am not very familiar with that. I am not sure if there is a generally accepted good practice for that. I know that we may have rounding issues when doing computations, but I don't think we do such computations with any of the fields of the Position.

What do you think? Is it good enough for now (to be modified if we ever do computations), or should I change it now? And in the latter case, what would be your suggestion? I've seen people using "epsilon"s, but I am not completely sure what it implies for e.g. latitudes and longitudes...

Reply:
After looking at it more closely, what you suggest is to implement a "close-enough" operator. Semantically, I've been told that `operator==` should actually be equal, and that I should use something else for "equal enough".

Reply:
I would suggest:
```
#include <limits>
return (abs(lhs.latitude_deg - rhs.latitude_deg) <= std::numeric_limits<T>::epsilon());
```